vsts/pipelines/templates/_buildimageBasesJobTemplate.yml

```diff
@@ -11,8 +11,6 @@ jobs:
   displayName: ${{ parameters.displayName }}
   timeoutInMinutes: 250
   steps:
-  - template: _cleanImageCacheTemplate.yml
-
```

Comment:
@kichalla i removed it, because I saw your Pr got closed and this needs to be removed from master